azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/MessageInfoModel.kt

```diff
@@ -56,7 +56,7 @@ internal fun com.azure.android.communication.chat.models.ChatMessageEditedEvent.
     return MessageInfoModel(
         internalId = null,
         id = this.id,
-        messageType = null,
+        messageType = ChatMessageType.TEXT,
```

Comment:
Should leave null

Reply:
because we don't want to over-ride the type (might be HTML)

Reply:
Ah... Thanks. I implemented the merge later but forgot to remove the type.